tests/framework/clients/rancher/generated/management/v3/zz_generated_aks_imported_cluster_config_spec.go

```diff
@@ -0,0 +1,50 @@
+package client
+
+const (
+	AKSImportedClusterConfigSpecType                          = "aksImportedClusterConfigSpec"
+	AKSImportedClusterConfigSpecFieldBalanceSimilarNodeGroups = "balanceSimilarNodeGroups"
+	AKSImortedClusterConfigSpecFieldDefaultRandStringLength   = "defaultRandStringLength"
+	AKSImortedClusterConfigSpecFieldDisableLocalAccounts      = "disableLocalAccounts"
+	AKSImortedClusterConfigSpecFieldDNSPrefix                 = "dnsPrefix"
+	AKSImortedClusterConfigSpecFieldEnableAutoScaling         = "enableAutoScaling"
+	AKSImortedClusterConfigSpecFieldEnablePublicIP            = "enablePublicIP"
+	AKSImortedClusterConfigSpecFieldEnableRBAC                = "enableRBAC"
+	AKSImortedClusterConfigSpecFieldKubernetesVersion         = "kubernetesVersion"
+	AKSImortedClusterConfigSpecFieldLoadProfileCount          = "loadProfileCount"
+	AKSImortedClusterConfigSpecFieldLocation                  = "location"
+	AKSImortedClusterConfigSpecFieldMaxCount                  = "maxCount"
+	AKSImortedClusterConfigSpecFieldMaxNodeProvisionTime      = "maxNodeProvisionTime"
+	AKSImortedClusterConfigSpecFieldMaxPods                   = "maxPods"
+	AKSImortedClusterConfigSpecFieldMinCount                  = "minCount"
```

Comment:
It seems like these are hardcoded cause of the typos in the const names. Is it the case? 

If this is hardcoded, these things will be gone whenever the v3 generation happens.